src/main/webapp/assets-lti/js/controllers/controllers.js

```diff
@@ -219,8 +219,9 @@ canvasSupportApp.controller('courseController', ['Course', 'Courses', 'Sections'
   TODO: MPATH dummy data now 
   adds to the scope a list if sections (by sis_section_id) that the current user can perform actions on
   */
+  var mPathwaysCoursesUrl = 'manager/mpathways/Instructors?instructor=bkg&termid=2060';
 
-  var mPathwaysCoursesUrl = 'assets-lti/data/mpathwaysdata.json';
+  //var mPathwaysCoursesUrl = 'assets-lti/data/mpathwaysdata.json';
```

Comment:
Also, is there a properties or configuration file that this should be in instead of in the code?